tests/UKHO.S100PermitService.Common.UnitTests/Handler/FakeHttpMessageHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Net;
+using System.Text;
+
+namespace UKHO.S100PermitService.Common.UnitTests.Handler
+{
+    public class FakeHttpMessageHandler : HttpMessageHandler
+    {
+        private readonly HttpResponseMessage _response;
+
+        public static HttpMessageHandler GetHttpMessageHandler(string content, HttpStatusCode httpStatusCode)
+        {
+            var response = new HttpResponseMessage()
+            {
+                StatusCode = httpStatusCode,
+                Content = new StringContent(content, Encoding.UTF8, "application/json")
+            };
+
+            var messageHandler = new FakeHttpMessageHandler(response);
+
+            return messageHandler;
+        }
+
+        public FakeHttpMessageHandler(HttpResponseMessage response)
+        {
+          _response = response;
+        }
+
+        protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
+        {
+            var tcs = new TaskCompletionSource<HttpResponseMessage>();
```

Comment:
use descriptive variable names

```suggestion
            var taskCompletionSource = new TaskCompletionSource<HttpResponseMessage>();
```

Reply:
Resolved